source/_docs/migrate-from-wpengine.md

```diff
@@ -5,28 +5,42 @@ categories: [wordpress]
 tags: [migrate]
 keywords: wordpress, pantheon, wpengine
 ---
-We recommend migrating WordPress sites from another host using the [Pantheon Migration](https://wordpress.org/plugins/bv-pantheon-migration/) plugin, developed by [BlogVault](https://blogvault.net/). However, sites can also be migrated manually.
-
-
-## Migrate Using the Pantheon Migration Plugin (Recommended)
-
-1. [Create a Pantheon account](https://dashboard.pantheon.io/register) or log in to your account.
-2. Create and install a new WordPress site on Pantheon:
- - Choose **Create a new site** from your Pantheon Dashboard.
- - Name your site.
- - Choose **WordPress** as your site type.
- - Wait for WordPress to be deployed and click **Visit Development Site** to complete the installation process.
-3. Create a [Pantheon machine token](https://pantheon.io/docs/machine-tokens/). Make sure the token has a descriptive name, including the name of your site on Pantheon. Save this machine token somewhere safe!
-  ![Create a machine token](/source/docs/assets/images/pantheon-create-machine-token.png)
-  <div class="alert alert-info" role="alert">
-  <h4>Note</h4>  
-  Earlier versions of the <a href="https://wordpress.org/plugins/bv-pantheon-migration/">Pantheon Migration</a> plugin request SFTP details instead of a Pantheon machine token. If prompted for SFTP credentials, please upgrade the plugin to the latest release.
-  </div>
-4. Right click on the **Visit Development Site** and select **Copy Link Address**.
-5. Go to your web host (source site), and install and activate the [Pantheon Migration](https://wordpress.org/plugins/bv-pantheon-migration/) plugin.
-6. Enter your Pantheon Dev site URL and Pantheon machine token you saved earlier.
-![BlogVault Plugin](/source/docs/assets/images/blogvault-setup-machine-token.png)
-7. Start the migration! You will receive an email, to the email address associated with your Pantheon account, when the migration begins and another email when the migration completes.
+We recommend migrating WordPress sites from another host using the [Pantheon Migration](https://wordpress.org/plugins/bv-pantheon-migration/) plugin, developed by [BlogVault](https://blogvault.net/).
+
+The following scenarios are exceptions to the recommended process and require [manually migrating](/docs/manual-import) the site:
+
+- Your site requires a custom upstream.
+- You would like to preserve the site's existing Git history.
+- [WordPress Site Networks](/docs/migrate-wordpress-site-networks)
+
+## Migrate Your Site to Pantheon
+
+From your Pantheon Dashboard:
+
+1. Choose **Migrate Existing Site**.
+2. Enter your current website URL.
+3. Select **WordPress**.
+4. Click **Continue**.
+5. Name your new Pantheon site.
+6. Select an organization for the site (optional).
+7. Click **Create Site**.
+8. Select **Generate Machine Token** and re-authenticate if prompted:
+
+ ![Authentication BlogVault migration](/source/docs/assets/images/migration-authentication-prompt.png)
```

Comment:
Can we move this error down to a troubleshooting section at the bottom? 

Also one other potential troubleshooting scenario (can you confirm?):
   - If you are logged in with one identity and re-authenticate with another account then the created site will be associated with one account and the machine token with another account, so when you attempt to start migration you'll get a "Destination site not found" error. We need to confirm if this can happen outside of @getpantheon and @pantheon.io accounts.